### PR TITLE
Document fallback timezone setting (fixes #2000)

### LINKED ISSUE #2000: [BUG] Invalid timezone in DSLR photos uploaded via CLI
### The bug

Just spent a few hours digging into why my DSLR photos were one hour off in time compared to my mobile photos.

Interestingly enough, this issue has plagued me over at google photos, too.

In the end, it was due to the microservice container having its timezone set to UTC.

But wait, we are using exif-vendored so this shouldn't be an issue? Alex himself says so here: https://github.com/immich-app/immich/issues/1519#issuecomment-1414953736

But alas, timezones still matter, because exif-vendored has to rely on local timezone information in some special cases, just like the DSLR photos.

The issue was finally solved for me by setting a timezone environment variable in the microservices container corresponding to my local timezone.

The obvious problem is that setting timezones in docker containers is messy, so it might not be an ideal solution.

Another solution is to move more of the timezone logic to somewhere that has native timezone information, like the CLI. Right now, the CLI uses a different exif library but I published a PR that makes the logic more consistent with immich itself:  too: https://github.com/immich-app/CLI/pull/69 

Note that the computed timezone data from the CLI is not even considered in [the current logic](https://github.com/immich-app/immich/blob/2ca560ebf80aadf1f7904bd88d7447d9a2a59ed3/server/apps/microservices/src/processors/metadata-extraction.processor.ts#L170) in most cases.

### The OS that Immich Server is running on

Ubuntu 22.04 via docker

### Version of Immich Server

1.50.1

### Version of Immich Mobile App

1.50.0 73

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

services:
  immich-server:
    container_name: immich_server
    image: altran1502/immich-server:release
    entrypoint: [ "/bin/sh", "./start-server.sh" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
    env_file:
      - .env
    environment:
      - NODE_ENV=production
      - TZ=Europe/Stockholm
    depends_on:
      - redis
      - database
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: altran1502/immich-server:release
    entrypoint: [ "/bin/sh", "./start-microservices.sh" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
    env_file:
      - .env
    environment:
      - NODE_ENV=production
      - TZ=Europe/Stockholm
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    image: altran1502/immich-machine-learning:release
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - model-cache:/cache
    env_file:
      - .env
    environment:
      - NODE_ENV=production
      - TZ=Europe/Stockholm
    restart: always

  immich-web:
    container_name: immich_web
    image: altran1502/immich-web:release
    entrypoint: [ "/bin/sh", "./entrypoint.sh" ]
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: redis:6.2
    restart: always

  database:
    container_name: immich_postgres
    image: postgres:14
    env_file:
      - .env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      PG_DATA: /var/lib/postgresql/data
    volumes:
      - pgdata:/var/lib/postgresql/data
    restart: always

  immich-proxy:
    container_name: immich_proxy
    image: altran1502/immich-proxy:release
    environment:
      # Make sure these values get passed through from the env file
      - IMMICH_SERVER_URL
      - IMMICH_WEB_URL
    ports:
      - 2283:8080
    logging:
      driver: none
    depends_on:
      - immich-server
    restart: always

volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
###################################################################################
# Database
###################################################################################

DB_HOSTNAME=immich_postgres
DB_USERNAME=postgres
DB_PASSWORD=postgres
DB_DATABASE_NAME=immich

# Optional Database settings:
# DB_PORT=5432

###################################################################################
# Redis
###################################################################################

REDIS_HOSTNAME=immich_redis

# Optional Redis settings:
# REDIS_PORT=6379
# REDIS_DBINDEX=0
# REDIS_PASSWORD=
# REDIS_SOCKET=

###################################################################################
# Upload File Config
###################################################################################

UPLOAD_LOCATION=/mnt/immich

###################################################################################
# Log message level - [simple|verbose]
###################################################################################

LOG_LEVEL=simple

###################################################################################
# Reverse Geocoding
####################################################################################

DISABLE_REVERSE_GEOCODING=false

# Reverse geocoding is done locally which has a small impact on memory usage
# This memory usage can be altered by changing the REVERSE_GEOCODING_PRECISION variable
# This ranges from 0-3 with 3 being the most precise
# 3 - Cities > 500 population: ~200MB RAM
# 2 - Cities > 1000 population: ~150MB RAM
# 1 - Cities > 5000 population: ~80MB RAM
# 0 - Cities > 15000 population: ~40MB RAM

REVERSE_GEOCODING_PRECISION=3

####################################################################################
# WEB - Optional
####################################################################################

# Custom message on the login page, should be written in HTML form.
# For example PUBLIC_LOGIN_PAGE_MESSAGE="This is a demo instance of Immich.<br><br>Email: <i>demo@demo.de</i><br>Password: <i>demo</i>"

PUBLIC_LOGIN_PAGE_MESSAGE="Photos"
```

### Reproduction steps

```bash
1. Take a JPG from a Nikon camera processed with Lightroom
2. Upload said JPG with immich-cli
3. Take a JPG from a mobile phone and upload it with either CLI or mobile app
4. Compare the dates reported by immich
...
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by etnoy**
This documentation improvement is a band-aid to fix #2000

### CODE REVIEW

File: docs/docs/FAQ.md [hunk 1]
```diff
@@ -16,10 +16,30 @@ sidebar_position: 7
...
 
+If the timezone is incorrect in an uploaded photo, check the ``DateTimeOriginal`` exif field of the uploaded file. Immich uses the very competent library [exiftool-vendored.js](https://github.com/photostructure/exiftool-vendored.js#dates) to handle timezone parsing, but in some cases (like photos taken with DSLR cameras) it has to fallback on the local timezone. If you are using docker, this fallback will be UTC. (Note that even the photo backup app that can't be named [has the same bug!](https://photo.stackexchange.com/a/126978)) In Immich, it is possible to change this assumed fallback timezone system-wide by setting the timezone in the microservices docker container. Note that you then must delete and re-upload the affected assets. As an example, the following modification of ```docker-compose.yml``` will set the timezone to be ``Europe/Stockholm``
```
Comment (jrasm91) [inline docs/docs/FAQ.md:23]: I don't think you need to delete and re-upload, just rerunning extract exif for all assets would fix it.

Comment (etnoy) [reply docs/docs/FAQ.md:23]: > I don't think you need to delete and re-upload, just rerunning extract exif for all assets would fix it.

Good point, will edit. Thanks!

**Comment by etnoy** [comment]
Done, now it should be fine to merge!

**Review by jrasm91 — APPROVED** [review]

File: docs/docs/FAQ.md [hunk 2]
```diff
@@ -16,10 +16,32 @@ sidebar_position: 7
...
+
+```
+  immich-microservices:
+    container_name: immich_microservices
+    image: altran1502/immich-server:release
+    entrypoint: [ "/bin/sh", "./start-microservices.sh" ]
+    volumes:
+      - ${UPLOAD_LOCATION}:/usr/src/app/upload
+    env_file:
+      - .env
```
[the remark this answers is not in the document]
Comment (etnoy) [reply docs/docs/FAQ.md]: Done

[the remark this answers is not in the document]
Comment (jrasm91) [reply docs/docs/FAQ.md]: Perfect. Thanks for the contribution!

**Pull request merged**